packages/jest-expo/src/preset/expoModules.js

```diff
@@ -801,7 +801,7 @@ module.exports = {
           debugMode: { type: 'boolean', mock: true },
           deviceName: { type: 'string' },
           deviceYearClass: { type: 'number', mock: 2019 },
-          executionEnvironment: { type: 'string' },
+          executionEnvironment: { type: 'string', mock: 'standalone' },
```

Comment:
when i added executionEnvironment to the mocks for the expo package i realized that we should probably do the same for our jest preset